### PR TITLE
Add security disclosure info

### PR DESCRIPTION
**Pull request by MarcDenman**
Added the standard security disclosure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,3 +110,7 @@ Update-AD $configData $true
 
 - Start small!
   - This module will generate a lot of changes in Active Directory which can make it very hard to verify the module is doing what you expect. Make the smallest possible change first, probably something like creating an OU or creating a group, ensure the change is what you wanted and then move onto the next step. Doing it all at once will make it nearly impossible to check it has been done correctly.
+
+## Security Disclosure
+
+The UK Hydrographic Office (UKHO) collects and supplies hydrographic and geospatial data for the merchant shipping and the Royal Navy, to protect lives at sea. Maintaining the confidentially, integrity and availability of our services is paramount. Found a security bug? You might be saving a life  by reporting it to us at UKHO-ITSO@gov.co.uk
```
Comment (elduddz) [inline README.md]: double space

**Review by elduddz — APPROVED** [review]

**Comment by thealmguy** [comment]
Is the email address right? Should it be UKHO-ITSO@ukho.gov.uk <mailto:UKHO-ITSO@ukho.gov.uk> ?

**Comment by MarcDenman** [comment]
Dunno. I was using the text sent to me. I suspect it isn't the correct email address, will check it tomorrow.

**Comment by elduddz** [comment]
I'm more interested in what happened with the commit message?

**Pull request merged**